Language.ml

```diff
@@ -369,6 +370,22 @@ let list = [
   shebangs = [{|lua|}];
   tags = [];
 };
+(*
+  Move language with SUI flavor
+*)
+{
+  id = Move_on_sui;
+  id_string = "move_on_sui";
+  name = "Move on Sui";
+  keys = [{|move_on_sui|}];
+  exts = [{|.move|}];
```

Comment:
@mjambon will this be problematic that we now have 2 languages (move_on_sui, move_on_aptos) using the same .move extension? Not sure how file targeting will work.